tools/config/manage-dependencies.ts

```diff
@@ -291,11 +292,37 @@ function filterLocalRelativeImports(
         return acc;
       }, {} as LibraryWithDependencies['tsImports']);
     lib.scssImports = Object.values(lib.scssImports)
-      .filter(
-        (imp) =>
-          imp.importPath.startsWith('node_modules/') ||
-          imp.importPath.startsWith('~')
-      )
+      .filter((imp) => {
+        if (
+          imp.importPath.startsWith('.') ||
+          imp.importPath.startsWith('url(')
+        ) {
+          return false;
+        }
+        if (imp.importPath.startsWith('@')) {
+          return true;
+        }
+        // whether imports can be resolved from relative path
+        let folder = path.dirname([...imp.files][0]);
+        let file;
+        if (imp.importPath.includes('/')) {
+          const parts = imp.importPath.split('/');
+          file = parts.pop();
+          folder = folder + '/' + parts.join('/');
```

Comment:
Instead of `parts.join('/')`, the foler path could perhaps be retrieved with `path.dirname(imp.importPath)`